database/sql/V057__add_fee_policy.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TYPE PolicyKind AS ENUM ('priceimprovement', 'volume');
+
+CREATE TABLE fee_policies (
+  auction_id bigint NOT NULL,
+  order_uid bytea NOT NULL,
+  -- The order in which the fee policies are inserted and applied.
+  application_order SERIAL NOT NULL,
+
+  -- The type of the fee policy.
+  kind PolicyKind NOT NULL,
+  -- The fee should be taken as a percentage of the price improvement. The value is between 0 and 1.
+  price_improvement_factor double precision,
+  -- Cap the fee at a certain percentage of the order volume. The value is between 0 and 1.
+  max_volume_factor double precision,
+  -- The fee should be taken as a percentage of the order volume. The value is between 0 and 1.
+  volume_factor double precision,
+
+  PRIMARY KEY (auction_id, order_uid, application_order)
```

Comment:
This automatically creates an index, right?
If we select by `auction_id` and `order_id` but **NOT** by `application_order` would we still hit this index or would we have to create an index with `(auction_id, order_id)`?

Reply:
The PK index is used while querying by `(auction_id, order_id)`. Adding a separate index here would probably be sufficient if we expect a lot of queries. Otherwise, it makes sense to keep it as it is to avoid additional disk usage.

Reply:
We have a test `postgres_roundtrip` that shows that by specifying the `(auction_id, order_id)` all rows are returned which is a desired behavior.